tests/phpunit/tests/functions/isBlogInstalled.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @since 6.0.0
+ *
+ * @group functions.php
+ * @covers ::is_blog_installed
+ */
+class Tests_Functions_IsBlogInstalled extends WP_UnitTestCase {
+
+	/**
+	 * Tests the default return value of is_blog_installed().
+	 * 
+	 * @ticket 54754
+	 */
+	public function test_is_blog_installed() {
+		$this->assertTrue( is_blog_installed() );
+	}
+
+	/**
+	 * @ticket 54754
+	 *
+	 * the other routes to true
+	 */
+	public function test_is_blog_installed_cache() {
+		// Prime the cache with true.
+		wp_cache_set( 'is_blog_installed', true );
+		$this->assertTrue( is_blog_installed(), 'cache set to true' );
```

Comment:
The `$message` argument for assertions should explain what went wrong, not what you're trying to do.

For example: `$this->assertTrue( $is_blog_installed, 'Did not respect the cached value of `true`' );`

🔢 Applies to other instances in the PR.

Reply:
changed